app/src/main/java/com/gh4a/fragment/SearchFragment.java

```diff
@@ -91,7 +94,7 @@ public static SearchFragment newInstance(int initialType, String initialQuery) {
     private static final String SUGGESTION_ORDER = SuggestionsProvider.Columns.DATE + " DESC";
 
     private static final String STATE_KEY_QUERY = "query";
-    private static final String STATE_KEY_SEARCH_TYPE = "search_type";
+    private static final String STATE_KEY_SEARCH_TYPE = SEARCH_TYPE;
 
```

Comment:
Extra key and state key are unrelated, please restore the string literal to not give the impression they *are* related.